lib/web_ui/dev/test_runner.dart

```diff
@@ -56,6 +56,17 @@ class TestCommand extends Command<bool> with ArgUtils {
             'at the same time. If this flag is set, only run the integration '
             'tests.',
       )
+      ..addFlag('use-system-flutter',
+        defaultsTo: false,
+        help: 'integration tests are using flutter repository for various tasks'
+        ', such as flutter drive, flutter pub get. If this flag is set, felt '
+        'will use flutter command without cloning the repository. This flag '
+        'can save internet bandwidth. However use with caution. Note that '
```

Comment:
Let's also recommend using this option when testing local changes in the Flutter framework. I imagine for most of contributors this will be the most common use-case.

Reply:
done.